rules/qc.smk

```diff
@@ -162,7 +162,9 @@ def get_descriptive_names(wildcards, input):
 
     labels = ""
     for file in input:
-        trep = re.findall(f"{wildcards.assembly}-([^\.]+)\.+?", file.split("/")[-1])[0]
```

Comment:
this truncated a name with a period in there. New code truncates after the `.sam` to capture both `.samtools` and `.sambamba`